UKHomeOffice/nmsw-ui
# Bug fix 404 error on FAL1 upload page

# Ticket



----

## BUG
A change to error handling meant the if condition that allowed a 404 for a GET declaration to continue on to the FAL1 upload form was failing.

**To replicate**
- click on create voyage

**Expected** 
- you are taken to the FAL1 upload page

**Actual**
- something went wrong shown

----

## To test

- check you can create a new voyage
- check once you've uploaded a FAL 1 (and nothing else) that you can upload a replacement FAL1
- check once you've uploaded Crew and/or passengers that you CANNOT upload a replacement FAL1
- - the task details link doesn't exist (it's just text)
- - if you go to /`report-voyage/upload-general-declaration?report=<decid>` it redirects to task details page

----

## Notes
